inline-translation/lib/original.js

```diff
@@ -96,6 +96,15 @@ function Original( original ) {
 		getComment: function() {
 			return comment;
 		},
+		getPlaceholders: function() {
+			var regexPattern = /%(\d\$)?([sd])/g;
+			var matchedPlaceholders = Array.from( singular.matchAll( regexPattern ) );
+			var placeholders = matchedPlaceholders.map( function( match ) {
+				return '<a class="inline-placeholder" href="#">' + match[ 0 ] + '</a>';
```

Comment:
Please add a checkmark if the translation is already used. This means that we need to run this function on keyup.